common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -98,6 +98,39 @@ The harmonic notch is designed to match the motor noise frequency as it changes
 
 .. _common-imu-notch-filtering-helicopter-setup:
 
+ESC based setup
+===============
+
+If ESCs with telemetry are used (either via a separate telemety wire or bi-directional dshot) then the harmonic notch reference frequency can be set dynamically using ESC telemetry.  The harmonic notch reference frequency parameter, :ref:`INS_HNTCH_FREQ <INS_HNTCH_FREQ>`, is used to indicate the lowest motor speed for which the ESC telemetry should be used to dynamically set the harmonic notch reference frequency.  It is recommended that this be set to below the hover frequency but above the control bandwidth.
```

Comment:
what does control bandwidth mean? you need to refer to a param value, I think....sched freq?

Reply:
Control bandwidth is the frequency at which the controller needs to operate and reflects the size and agility of the copter. Thus for a big, heavy copter the control bandwidth might be 30Hz, for small racer it might be 80Hz. Maybe we could say that a good rule of thumb is that this should be above the configured value for INS_GYRO_FILTER. Basically we don't want the notch filtering either stick or control inputs.